File: archive/2025-11-old-scripts/take-homepage-screenshot.py
"""
Take full-page screenshot of homepage
"""
from playwright.sync_api import sync_playwright
import os

def take_screenshot():
    screenshots_dir = r"C:\Users\denit\Local Sites\svetlinkielementor\screenshots"
    os.makedirs(screenshots_dir, exist_ok=True)

    with sync_playwright() as p:
        browser = p.chromium.launch()
        page = browser.new_page(viewport={'width': 1920, 'height': 1080})

        print("Navigating to homepage...")
        page.goto('http://svetlinkielementor.local/home')
        page.wait_for_load_state('networkidle')

        screenshot_path = os.path.join(screenshots_dir, 'homepage-current-full.png')
        print("Taking full-page screenshot...")
        page.screenshot(path=screenshot_path, full_page=True)

        browser.close()
        print(f"Screenshot saved to: {screenshot_path}")

if __name__ == "__main__":
    take_screenshot()

File: archive/2025-11-old-scripts/take-homepage-screenshot.js
/**
 * Take full-page screenshot of homepage
 */
const { chromium } = require('playwright');
const fs = require('fs');
const path = require('path');

async function takeScreenshot() {
    const screenshotsDir = path.join(__dirname, 'screenshots');
    if (!fs.existsSync(screenshotsDir)) {
        fs.mkdirSync(screenshotsDir, { recursive: true });
    }

    const browser = await chromium.launch();
    const page = await browser.newPage({ viewport: { width: 1920, height: 1080 } });

    console.log('Navigating to homepage...');
    await page.goto('http://svetlinkielementor.local/home');
    await page.waitForLoadState('networkidle');

    const screenshotPath = path.join(screenshotsDir, 'homepage-current-full.png');
    console.log('Taking full-page screenshot...');
    await page.screenshot({ path: screenshotPath, fullPage: true });

    await browser.close();
    console.log(`Screenshot saved to: ${screenshotPath}`);
}

takeScreenshot().catch(console.error);
